### PR TITLE
second assignments completed

### PR DESCRIPTION
**Pull request by SchruteFarmsCode**
own implementation of stack

### CODE REVIEW

File: src/main/java/Main.java
Comment (pmat2) [inline src/main/java/Main.java:1]: no need to change main

File: src/main/java/fundamentals/exception/TextTooLongException.java [hunk 1]
```diff
@@ -3,6 +3,5 @@
 public class TextTooLongException extends Exception {
     public TextTooLongException(String text) {
         super(text);
-
```
Comment (pmat2) [inline src/main/java/fundamentals/exception/TextTooLongException.java:6]: cleaning up should not be addressed in this PR

File: src/main/java/fundamentals/exception/TextTooShortException.java [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 public class TextTooShortException extends Exception {
     public TextTooShortException(String text) {
-
```
Comment (pmat2) [inline src/main/java/fundamentals/exception/TextTooShortException.java:5]: cleaning up should not be addressed in this PR

**Review by pmat2 — CHANGES_REQUESTED** [review]

**Comment by pmat2** [comment]
> _ No description provided. _

Fill description

File: src/main/java/fundamentals/MyStackImplementation.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package fundamentals;
+
+import java.util.ArrayList;
+import java.util.EmptyStackException;
+import java.util.List;
+
+public class MyStackImplementation {
```
Comment (pmat2) [inline src/main/java/fundamentals/MyStackImplementation.java]: This needs to be generic

File: src/main/java/fundamentals/MyStackImplementation.java
Comment (pmat2) [inline src/main/java/fundamentals/MyStackImplementation.java:1]: move to new package `collections`

Comment (pmat2) [inline src/main/java/fundamentals/MyStackImplementation.java:1]: Rename to MyStack

File: src/main/java/Main.java
Comment (SchruteFarmsCode) [reply src/main/java/Main.java:1]: corrected

**Review by pmat2 — CHANGES_REQUESTED** [review]

File: Assignments.iml
Comment (pmat2) [inline Assignments.iml:1]: this should not be in remote repository

File: src/main/java/Main.java
Comment (pmat2) [reply src/main/java/Main.java:1]: Not really 
image

**Review by pmat2 — CHANGES_REQUESTED** [review]

**Review by pmat2 — APPROVED** [review]

**Pull request merged**